web/themes/dcamp_2017_theme/templates/page.html.twig

```diff
@@ -56,9 +56,13 @@
       {{ page.content }}
     </div>
   </main>
+
+  <div id="floating-button">
+    <a class="button" href="https://www.eventbrite.es/e/entradas-drupalcamp-spain-2017-33218432225" target="_blank">{% trans %}Buy ticket{% endtrans %}</a>
```

Comment:
This is way too intrusive. Specially on mobile devices:

![image](https://cloud.githubusercontent.com/assets/108130/24646177/47e79334-191b-11e7-8237-5bf277e49e36.png)

Can we remove it? I would rather rely on promoting this via social networks.